src/net_processing.h

```diff
@@ -217,6 +217,9 @@ class PeerManager final : public CValidationInterface, public NetEventsInterface
     /** Send a version message to a peer */
     void PushNodeVersion(CNode& pnode, int64_t nTime);
 
+    /** Send a ping message on a regular schedule or if requested manually */
+    void MaybeSendPing(CNode* pto);
```

Comment:
Just so that the commits are hygienic. For people reviewing commit by commit, it wouldn't make sense to have a function that returns a bool which is just ignored. Changing the return type in a subsequent commit is as easy as changing void to bool in the declaration and definition.

I was considering not returning a bool here, since it's a bit of a strange interface (the function sets `pto.fDisconnect` _and_ returns false. Perhaps it should stay as returning `void`, and have `SendMessages()` check for `fDisconnect` after calling this function?